### PR TITLE
HPCC-12066 Document FROMJSON

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia jamesdefabia@lexisnexis.com
@afishbeck Please review

### CODE REVIEW

File: docs/ECLLanguageReference/ECLR_mods/BltInFunc-FROMJSON.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+  <para><informaltable colsep="1" frame="all" rowsep="1">
+      <tgroup cols="2">
+        <colspec colwidth="77.95pt" />
+
+        <tbody>
+          <row>
+            <entry><emphasis>record</emphasis></entry>
+
+            <entry>The RECORD structure to produce. Each field must specify
```
Comment (afishbeck) [inline docs/ECLLanguageReference/ECLR_mods/BltInFunc-FROMJSON.xml]: Technically "should" specify xpath.  No XPATH means use the field name lower case.

File: docs/ECLLanguageReference/ECLR_mods/BltInFunc-FROMJSON.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  STRING10 Firstname{xpath('FName')};  
+  STRING10 Lastname{xpath('LName')}; 
+END; 
+x := '{"FName": "George" , "LName": "Jetson", "EmpID": 42}'; 
+rec := FROMJSON(namesRec,x); 
+OUTPUT(rec);
+</programlisting>
+
+  <para>See Also: <link linkend="ROW">ROW</link>, <link
+  linkend="TOXML">TOJSON</link></para>
```
Comment (afishbeck) [inline docs/ECLLanguageReference/ECLR_mods/BltInFunc-FROMJSON.xml]: Not sure what effect the linkend should probably be "TOJSON"

Comment (JamesDeFabia) [inline docs/ECLLanguageReference/ECLR_mods/BltInFunc-FROMJSON.xml]: good catch!

**Comment by afishbeck** [comment]
@JamesDeFabia looks good

**Pull request merged**